packages/cardano-services/src/cli.ts

```diff
@@ -175,24 +172,28 @@ withCommonOptions(
   );
 
 withCommonOptions(
-  withOgmiosOptions(
-    withPostgresOptions(
+  withHandlePolicyIdsOptions(
```

Comment:
As minor as above, but here is even more true

Reply:
af17ac42b57672c085e8d89d7f2565a979aaad0c should fix this